.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,24 @@
+name: Pull Requst
```

Comment:
typo: Request
```suggestion
name: Pull Request
```

Reply:
Fixed in #8 that's a prerequisite of this MR thanks!